sentry-kotlin-compiler-plugin/build.gradle.kts

```diff
@@ -76,3 +90,20 @@ plugins.withId("com.vanniktech.maven.publish.base") {
     }
   }
 }
+
+tasks.withType<Jar> {
+  from(kotlin1920.output)
+  from(kotlin2120.output)
+}
+
+// see
+// https://youtrack.jetbrains.com/issue/KTIJ-24311/task-current-target-is-17-and-kaptGenerateStubsProductionDebugKotlin-task-current-target-is-1.8-jvm-target-compatibility-should
+kotlin { jvmToolchain(11) }
+
+tasks.withType<KotlinCompile>().configureEach {
+  kotlinOptions {
+    jvmTarget = JavaVersion.VERSION_11.toString()
+    languageVersion = "1.9"
```

Comment:
not sure if it has to be a breaking change, but probably impact is quite lot as it's nearly 2 years old.

Reply:
I guess it's fine, we just have to check if the compiler plugin is going to work on kotlin < 1.9 with this options? I assume it probably should